Assets/Scenes/Main.unity

```diff
@@ -1867,6 +1868,67 @@ PrefabInstance:
       propertyPath: targetGameObject
       value: 
       objectReference: {fileID: 767208734}
+    - target: {fileID: 6178400463341988642, guid: 5fbb7c7c8ec47e849afba1b658f3e07e,
```

Comment:
Klopt het dat ik hier een rits aan functionalities zie? Er is er nu toch maar 1 bijgekomen? Of lees ik dit verkeerd?